tensorrt_llm/llmapi/llm_args.py

```diff
@@ -279,6 +279,48 @@ class MoeConfig(StrictBaseModel):
         "Use low precision combine in MoE operations (only for NVFP4 quantization). When enabled, uses lower precision for combining expert outputs to improve performance."
     )
 
+    # moe weight prefetching config
+    use_prefetch: bool = Field(default=False,
```

Comment:
`use_prefetch` can be seen as `prefetch_config is not None`. Let's remove this field.